propkatraj/tests/test_regression.py

```diff
@@ -275,13 +178,4 @@ def test_nonprotein_residues(tmpdir):
     with tmpdir.as_cwd():
         wmsg = "Non protein atoms passed"
         with pytest.warns(UserWarning, match=wmsg):
-            pkas = propkatraj.get_propka(u, sel="not protein", stop=1)
-
-
-def test_nonprotein_residues_analysisbase(tmpdir):
-    """Test for Issue #24 - remove once PDBWriter is fixed"""
-    u = mda.Universe(GRO, XTC)
-    with tmpdir.as_cwd():
-        wmsg = "Non protein atoms passed"
-        with pytest.warns(UserWarning, match=wmsg):
-            pkatraj = propkatraj.PropkaTraj(u, select="not protein")
```

Comment:
Why is this line being changed?

Reply:
@IAlibay style enforcement, [flake8](https://www.flake8rules.com/rules/F841.html). Alternatively we could also just not assign it?

Reply:
I see, I don't particularly mind either way. I generally just don't assign since it's a test for a warning, but also "whatever works" is fine here.